tests/acceptance/bootstrap/SearchContext.php

```diff
@@ -220,4 +218,75 @@ public function theSearchResultShouldContainEntriesWithHighlight(
 			);
 		}
 	}
+
+	/**
+	 * @When /^user "([^"]*)" searches for "([^"]*)" inside (folder|space) "([^"]*)"(?: in space "([^"]*)")? using the WebDAV API$/
+	 *
+	 * @param string $user
+	 * @param string $pattern
+	 * @param string $scope
+	 * @param string|null $scopeName
+	 * @param string|null $spaceName
+	 *
+	 * @return void
+	 */
+	public function userSearchesInsideFolderOrSpaceUsingWebDavAPI(
+		string $user,
+		string $pattern,
+		string $scope,
+		string $scopeName = null,
+		string $spaceName = null
+	): void {
+		// NOTE: since indexing of newly uploaded files or directories with ocis is decoupled and occurs asynchronously, a short wait is necessary before searching
```

Comment:
it is too long
```suggestion
		// NOTE: since indexing of newly uploaded files or directories with ocis is decoupled and occurs asynchronously,
		// a short wait is necessary before searching
```